contributing.md

```diff
@@ -64,6 +64,11 @@ npm run build
 npm run build -- --scope <package-name>
 ```
 
+### Build all libraries
+
+Link the libraries toghether: from the root of the repo `npx lerna bootstrap`
```

Comment:
This should be handled automatically by a postinstall script when you install from the root. We're also planning to ditch lerna sometime in the near future so we should stick to just `npm install`

Reply:
Ok, reverted